src/lib/icons/audio/VolumeIcon.svelte

```diff
@@ -1,8 +1,13 @@
 <script lang="ts">
+    // Volume from https://fonts.google.com/icons
+    // Filled = 1
+    // Weight = 400
+    // Grade = 0
+    // Optical size = 24px
     export let _class = 'fill-black-100';
 </script>
 
-<svg class={_class} xmlns="http://www.w3.org/2000/svg" height="20" width="20"
+<svg class={_class} xmlns="http://www.w3.org/2000/svg" height="24" width="24"
```

Comment:
The data on this one is the same, but just the height and width attributes where changed.  Please regenerate the icon with the optical size = 24px.